bundles/binding/org.openhab.binding.weather/src/main/java/org/openhab/binding/weather/internal/common/LocationConfig.java

```diff
@@ -24,7 +24,7 @@
     private Double latitude;
     private Double longitude;
     private String woeid;
-    private Integer updateInterval;
+    private Integer updateInterval = 240;
```

Comment:
Yes; that is what I mean.  Doing it this way has value to future developers who want to understand where the magic number 240 came from.  By explicitly describing it the way you propose, any doubt is removed as to what the significance of 240 is.  [This stackexchange](https://stackoverflow.com/questions/66066/what-is-the-best-way-to-implement-constants-in-java) is a bit all over the place, but the "consensus" there is that constants expressed this way help explain the intention.  Also, the compiler will effectively produce the same bytecode anyway.  You could make it `public` instead of `private` if it was needed from another class, but since it's `static` and `final`, it would be just as safe.    In general, it's better to write verbose code like this so it's easier for others to understand your intentions.